packages/breadcrumbs/src/styled/StyledBreadcrumbItem.ts

```diff
@@ -6,23 +6,44 @@
  */
 
 import styled from 'styled-components';
-import classNames from 'classnames';
-import { retrieveComponentStyles } from '@zendeskgarden/react-theming';
-import BreadcrumbStyles from '@zendeskgarden/css-breadcrumbs';
+import stripUnit from 'polished/lib/helpers/stripUnit';
+import { getColor } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'breadcrumbs.item';
 
 export interface IStyledBreadcrumbItemProps {
   isCurrent?: boolean;
 }
 
-export const StyledBreadcrumbItem = styled.li.attrs<IStyledBreadcrumbItemProps>(props => ({
+/**
+ * 1. CSS targets Microsoft Edge
+ */
+const anyLinkStyles = (styles: string) => `
+  & a {
```

Comment:
This is not the current styling seen in the CSS https://unpkg.com/browse/@zendeskgarden/css-breadcrumbs@0.2.20/dist/index.css. If you are styling for a child `a` tag, then none of the `:any-link` pseudo is necessary. The point of `:any-link` is to apply styling to any "link-ish" type element. See https://developer.mozilla.org/en-US/docs/Web/CSS/:any-link for details.